doc/source/user_guide/getting_help.rst

```diff
@@ -28,6 +28,24 @@ The ``quick search`` field located in the navigation bar of the html
 documentation can be used to search for specific keywords (segmentation,
 rescaling, denoising, etc.).
 
+API Discovery
+-------------
+
+The users can make use of ``np.lookfor`` method to check the functions 
```

Comment:
Hmm... I don't think that it is a good idea to call it `method`. `function` should be better.